src/services/swap_service.ts

```diff
@@ -602,6 +612,31 @@ export class SwapService {
             decodedUniqueId,
         };
     }
+
+    private async _getAltMarketOfferingsAsync(): Promise<AltRfqtMakerAssetOfferings> {
+        if (!this._altRfqMarketsCache) {
+            this._altRfqMarketsCache = createResultCache<AltRfqtMakerAssetOfferings>(async () => {
+                if (ALT_RFQ_MM_ENDPOINT === undefined || ALT_RFQ_MM_API_KEY) {
```

Comment:
is this supposed to be `ALT_RFQ_MM_API_KEY === undefined` ?

Reply:
yep woops